Mathlib/RingTheory/Flat/Basic.lean

```diff
@@ -273,6 +278,486 @@ lemma iff_lTensor_preserves_injective_linearMap [Small.{v} R] :
       (L : N →ₗ[R] N'), Function.Injective L → Function.Injective (L.lTensor M) := by
   simp_rw [iff_rTensor_preserves_injective_linearMap, LinearMap.lTensor_inj_iff_rTensor_inj]
 
+variable (R M) in
+lemma lTensor_exact [Small.{v} R] [flat : Flat R M] ⦃N N' N'' : Type v⦄
```

Comment:
Is this `TensorProduct.lTensor_exact` in `LinearAlgebra/TensorProduct/RightExactness.lean`?

Reply:
Almost, but in `RightExactness`, it requires surjectivity. So this is showing tensoring is not only right exact, but also left exact.